python/GafferSceneUI/TranslateToolUI.py

```diff
@@ -51,6 +53,8 @@
 	"viewer:shortCut", "W",
 	"order", 1,
 
+	"ui:tool:transformToolTips", IECore.StringVectorData( [ "Hold 'V' and click to move to target", ] ),
```

Comment:
I think "snap to geometry" would have a more familiar ring than "move to target" for most people.

Reply:
Changed in https://github.com/GafferHQ/gaffer/pull/5558/commits/ed4eb1588f632803363529cdc831140ffed18f4c